docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,197 @@
+        }
+      }
+
+      depends_on = [aiven_kafka_connect.kc,aiven_pg.demo]
+    }
+
+    resource "aiven_kafka_connector" "cdc-connector" {
+      project        = var.project
+      service_name   = aiven_kafka_connect.kc.service_name
+      connector_name = "kafka-pg-source"
+
+      config = {
+        "name"            = "kafka-pg-source"
+        "connector.class" = "io.debezium.connector.postgresql.PostgresConnector",
+        "snapshot.mode"   = "initial"
+        "database.hostname" : aiven_pg.demo.service_host
+        "database.port" : aiven_pg.demo.service_port
+        "database.password" : aiven_pg.demo.service_password
+        "database.user" : aiven_pg.demo.service_username
+        "database.dbname"           = "defaultdb"
+        "database.server.name"      = "replicator"
+        "database.ssl.mode"         = "require"
+        "include.schema.changes"    = true
+        "include.query"             = true
+        "table.include.list"        = "public.tab1"
+        "plugin.name"               = "wal2json"
+        "decimal.handling.mode"     = "double"
+        "_aiven.restart.on.failure" = "true"
+        "heartbeat.interval.ms"     = 30000
+        "heartbeat.action.query"    = "INSERT INTO heartbeat (status) VALUES (1)"
+      }
+      depends_on = [aiven_service_integration.i1,aiven_kafka_connect.kc,aiven_pg.demo]
+    }
+
+Let's see the different resources we are going to create:
+
+- Version 3.2.1 of the Aiven Terraform provider will be used
+- The PostgreSQL database will be created in "google-europe-north1" cloud provider with a business-4 plan
+- The Aiven Apache Kafka service will be created in "azure-norway-west" cloud and will be preconfigured with a number of properties:
+  
+  - The ``auto_create_topics_enable = true`` property is crucial as it allows the Debezium connector to create the Kafka topics directly.
+  - The ``kafka_connect = false`` property is needed because we want to create a separate Aiven Apache Kafka Connect service.
+
+
+- One Aiven Apache Kafka Connect service is configured with public access
+- Then a service integration is created within Kafka Connect service. This integration will use 2 internal topics for storing status and offset.
+- The last Aiven service that will be provisioned is the actual Debezium source connector for PostgreSQL, which is specified by the "connector.class" and is configured with the connection strings to access the PostgreSQL database and listen for all data changes on one or more tables. In our case, it will be "tab1" in "defaultdb", "public" schema. The plugin used is "wal2json" that converts WAL events (WAL stands for Write Ahead Logging) into json payload that is sent to the Kafka topic. The Kafka topic that the Debezium connector creates has the name "replicator.public.tab1", where "replicator" is the logical database used by Debezium connector to monitor for data changes and "public" and "tab1" are the name of the schema and the table name respectively.
```

Comment:
A general note for all Vale warnings. Let's try to fix them. For example, "JSON" instead of "json", ``defaultdb`` instead of "defaultdb", etc. Please refer to https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst#formatting-guidelines for details. I'm happy to assist with any warning that you might be stuck with.